src/hotspot/share/gc/shared/gc_globals.hpp

```diff
@@ -677,15 +677,15 @@
   product(uint, MarkSweepAlwaysCompactCount,     4,                         \
           "How often should we fully compact the heap (ignoring the dead "  \
           "space parameters)")                                              \
-          range(1, max_juint)                                               \
+          range(1, UINT_MAX)                                                \
                                                                             \
   develop(uintx, GCExpandToAllocateDelayMillis, 0,                          \
           "Delay between expansion and allocation (in milliseconds)")       \
                                                                             \
-  product(uintx, GCDrainStackTargetSize, 64,                                \
+  product(size_t, GCDrainStackTargetSize, 64,                               \
           "Number of entries we will try to leave on the stack "            \
           "during parallel gc")                                             \
-          range(0, max_juint)                                               \
+          range(0, SIZE_MAX)                                                \
```

Comment:
This is a large increase in the max on 64bit platforms.

Reply:
Yes, you should revert the change from max_juint to SIZE_MAX.  Is the change to size_t a cleanup to remove a cast?

Reply:
Reverted back to original.